### PR TITLE
[Snyk] Fix for 2 vulnerabilities

### PR DESCRIPTION
**Pull request by SiAdcock**
<p>This PR was automatically created by Snyk using the credentials of a real user.</p><br /><h3>Snyk has created this PR to fix one or more vulnerable packages in the `yarn` dependencies of this project.</h3>

#### Changes included in this PR

- Changes to the following files to upgrade the vulnerable dependencies to a fixed version:
    - package.json

<details>
<summary>⚠️ <b>Warning</b></summary>

```
Failed to update the yarn.lock, please update manually before merging.
```
</details>

##### With an upgrade:
Severity                   | Priority Score (*)                   | Issue                   | Breaking Change                   | Exploit Maturity
:-------------------------:|-------------------------|:-------------------------|:-------------------------|:-------------------------
medium severity  |  **646/1000**  <br/> **Why?** Proof of Concept exploit, Has a fix available, CVSS 6.5  | Prototype Pollution <br/>[SNYK-JS-DSET-2330881](https://snyk.io/vuln/SNYK-JS-DSET-2330881) |  Yes  | Proof of Concept 
medium severity  |  **646/1000**  <br/> **Why?** Proof of Concept exploit, Has a fix available, CVSS 6.5  | Cross-site Scripting (XSS) <br/>[SNYK-JS-VIDEOJS-1533429](https://snyk.io/vuln/SNYK-JS-VIDEOJS-1533429) |  Yes  | Proof of Concept 

(*) Note that the real score may have changed since the PR was raised.

Check the changes in this PR to ensure they won't cause issues with your project.

------------

**Note:** *You are seeing this because you or someone else with access to this repository has authorized Snyk to open fix PRs.*

For more information:  
🧐 [View latest project report](https://app.snyk.io/org/the-guardian/project/bf45c888-22f4-49e4-8a4f-d5b4f81db764?utm_source&#x3D;github&amp;utm_medium&#x3D;referral&amp;page&#x3D;fix-pr)

🛠 [Adjust project settings](https://app.snyk.io/org/the-guardian/project/bf45c888-22f4-49e4-8a4f-d5b4f81db764?utm_source&#x3D;github&amp;utm_medium&#x3D;referral&amp;page&#x3D;fix-pr/settings)

📚 [Read more about Snyk's upgrade and patch logic](https://support.snyk.io/hc/en-us/articles/360003891078-Snyk-patches-to-fix-vulnerabilities)

[//]: # (snyk:metadata:{"prId":"c1e129c2-46e4-4660-ad66-e1eda54531a9","prPublicId":"c1e129c2-46e4-4660-ad66-e1eda54531a9","dependencies":[{"name":"prebid.js","from":"6.26.0","to":"7.15.0"},{"name":"video.js","from":"5.3.0","to":"7.14.3"},{"name":"videojs-ima","from":"0.3.0","to":"0.7.0"}],"packageManager":"yarn","projectPublicId":"bf45c888-22f4-49e4-8a4f-d5b4f81db764","projectUrl":"https://app.snyk.io/org/the-guardian/project/bf45c888-22f4-49e4-8a4f-d5b4f81db764?utm_source=github&utm_medium=referral&page=fix-pr","type":"auto","patch":[],"vulns":["SNYK-JS-DSET-2330881","SNYK-JS-VIDEOJS-1533429"],"upgrade":["SNYK-JS-DSET-2330881","SNYK-JS-VIDEOJS-1533429"],"isBreakingChange":true,"env":"prod","prType":"fix","templateVariants":["pr-warning-shown","priorityScore"],"priorityScoreList":[646,646]})

---

**Learn how to fix vulnerabilities with free interactive lessons:**

 🦉 [Prototype Pollution](https://learn.snyk.io/lessons/prototype-pollution/javascript/?loc&#x3D;fix-pr)
 🦉 [Cross-site Scripting (XSS)](https://learn.snyk.io/lessons/xss/javascript/?loc&#x3D;fix-pr)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -39,14 +39,14 @@
 		"object-fit-videos": "^1.0.3",
 		"ophan-tracker-js": "1.4.0",
 		"preact": "^10.5.13",
-		"prebid.js": "https://github.com/guardian/Prebid.js#0e9f87a",
+		"prebid.js": "7.15.0",
```
Comment (mxdvl) [inline package.json:42]: This cannot be right. We have our own custom build of Prebid. cc. @guardian/commercial-dev

File: package.json [hunk 2]
```diff
@@ -39,14 +39,14 @@
...
 		"qwery": "3.4.2",
 		"rangefix": "^0.2.5",
 		"raven-js": "^3.19.1",
-		"video.js": "~5.3",
+		"video.js": "~7.14.3",
 		"videojs-contrib-ads": "3.1.3",
 		"videojs-embed": "guardian/videojs-embed#v0.3.3",
-		"videojs-ima": "googleads/videojs-ima#0.3.0",
+		"videojs-ima": "0.7.0",
```
Comment (mxdvl) [inline package.json:49]: Why are we moving away from `googleads`’s version?

**Comment by bryophyta** [comment]
This has been open for a while, does anyone know what it's status is? (cc. @SiAdcock & @mxdvl )

**Comment by bryophyta** [comment]
Bumping again -- is this a live PR/issue, or has it been resolved in another PR in the meantime? (cc. @SiAdcock & @mxdvl)

**Comment by mxdvl** [comment]
The Prebid update cannot be right. I’ll let @guardian/commercial-dev make a call, but I think this PR is now way out of date.

**Comment by SiAdcock** [comment]
Sorry folks, missed this. Didn't even know I'd raised it TBH! It looks... wrong. I'm gonna close.

**Pull request closed without merging**